code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -152,6 +152,16 @@
 	modifier = 5
 	mix_message = "<span class='boldannounce'>Sparks start flying around the gunpowder!</span>"
 
+/datum/chemical_reaction/anfo
+	results = list(/datum/reagent/anfo = 3)
+	required_reagents = list(/datum/reagent/plantnutriment/eznutriment = 1, /datum/reagent/plantnutriment/left4zednutriment = 1, /datum/reagent/plantnutriment/robustharvestnutriment = 1, /datum/reagent/fuel = 1)
+
+/datum/chemical_reaction/reagent_explosion/anfo_explosion
+	required_reagents = list(/datum/reagent/anfo = 1)
+	required_temp = 474
+	strengthdiv = 5
+	modifier = 10
```

Comment:
these should be moved below the gunpowder_explosion/on_reaction proc.

Reply:
thank you i didnt see the proc